### DISCUSSION TITLE
Ended 3 Pro with EZABL

### DISCUSSION THREAD

**Discussion by LakerNetman** [jneilliii/OctoPrint-BedLevelVisualizer#667] [Q&A] [answered]
Hi. Has anyone used BLV with an Ender 3 that has an EZABL auto leveler? Any suggestions on settings to tweak in the sample code blocks would be appreciated.

**Comment by jneilliii** [comment, marked as answer]
ez-abl is just a probe, the type of leveling is based on your firmware and what you enabled in it for bed leveling.

**Reply by LakerNetman** [reply]
Thanks. You pointed me in the right direction. I read through all of the Marlin docs related to bed leveling and have everything working now.
